### DISCUSSION TITLE
Multiple URL/FIle Links in Nodes

### DISCUSSION THREAD

**Discussion by GenerallyDisjointed** [freeplane/freeplane#452] [Help]
Is there a way I can add multiple links in a node? At the moment I link a document, but want extend that by adding other files and URL's

**Comment by EdoFro** [comment]
you can add them as attributes.

1. add a new attribute
2. right click on the attribute
   - popup menu appears
   - click on **"Add hyperlink (URL) ..."**

**Reply by EdoFro** [reply]
examples:
![image](https://user-images.githubusercontent.com/35700575/173396671-ed7532e4-13ec-4937-b2d3-af0ec3b93c61.png)

or in the ToolPanel:
![image](https://user-images.githubusercontent.com/35700575/173396853-0dec5c59-a497-45c5-a3b7-f820bd3fe879.png)

paste the url:
![image](https://user-images.githubusercontent.com/35700575/173397059-ce6a8b6c-ed59-41af-a98a-1ac5dc61bce7.png)

now it has a link as attribute:
![image](https://user-images.githubusercontent.com/35700575/173397277-f2b280f5-2ac1-47b0-b478-1ffd79c09900.png)

**Comment by GenerallyDisjointed** [comment]
Fantastic - thankyou for your help!
